src/Controller/EnMarche/NewsletterController.php

```diff
@@ -31,6 +36,14 @@ public function subscriptionAction(Request $request)
         if ($form->isSubmitted() && $form->isValid()) {
             $this->get('app.newsletter_subscription.handler')->subscribe($subscription);
 
+            if ($path = $newsletterSubscriptionProcess->getSuccessRedirectPath() ?? null) {
+                $newsletterSubscriptionProcess->terminate();
+
+                $this->addFlash('info', $translator->trans('newsletter.subscription.success'));
```

Comment:
flash message is translated in base.html.twig, you can use the key here

Reply:
Nice catch! thx you :+1: